storage-proofs-post/src/election/vanilla.rs

```diff
@@ -67,7 +65,7 @@ pub struct PublicInputs<T: Domain> {
     pub prover_id: T,
     #[serde(bound = "")]
     pub comm_r: T,
-    pub partial_ticket: Fr,
+    pub partial_ticket: [u8; 32],
```

Comment:
Note: changing `partial_ticket`'s type from `Fr` to `[u8; 32]` (rather than a generic field `F`) makes the trait bounds much nicer.